tc-client/src/main/java/com/tc/object/DistributedObjectClient.java

```diff
@@ -264,7 +256,9 @@ private synchronized ClientMessageChannel internalStart(int socketTimeout) {
                                                                  socketTimeout);
 
 
-    final ClientIDLoggerProvider cidLoggerProvider = new ClientIDLoggerProvider(clientChannel::getClientID);
+    WeakReference<ClientMessageChannel> channelRef = new WeakReference<>(clientChannel);
+
```

Comment:
Yuck...should we just get rid of the special logger?  Or maybe use logback or slf4j conventions to achieve something similar?